### DISCUSSION TITLE
Copilot: Node.js version 12.x–17.x required but found 18.1.0

### DISCUSSION THREAD

**Discussion by 127** [community/community#16927] [Projects and Issues] [answered]
os x 12.4 + homebrew  node 18.1.0
fails to run 
Error: Copilot: Node.js version 12.x–17.x required but found 18.1.0

**Comment by omerbensaadon** [comment, marked as answer]
Hey @127 this is not relevant to GitHub Issues, I would post this in https://github.com/orgs/github-community/discussions/categories/copilot

**Comment by omertuc** [comment]
See discussion https://github.com/orgs/community/discussions/16800

**Reply by Asd19101993Asd** [reply]
Gft

**Reply by Asd19101993Asd** [reply]
Tt
